web/docs/tql2/functions/otherwise.md

```diff
@@ -0,0 +1,39 @@
+# otherwise
+
+Returns a `fallback` value if `primary` is `null`.
+
+```tql
+otherwise(primary:expr, fallback:expr)
+```
+
+## Description
+
+The `otherwise` function evaluates its arguments and replaces `primary` with
+`fallback` where `primary` would be `null`.
+
+### `primary: expr`
+
+The expression to return if not `null`.
+
+### `fallback: expr`
+
+The expression to return if `primary` evaluated to `null`.
+
+## Examples
+
+### Set a default value for a key
+
+```tql
+from [
+  {x: 1},
+  {x: 2},
+  {}
+]
+x = x.otherwise(-1)
```

Comment:
I'm also not the biggest fan of the long name, this could be considered a common operation. Is the intention to have something temporary until we have for example 1st class syntax for that, or what is the plan here?

Reply:
I'm open to suggestions. I think the name is readable and explanatory to get this out first.

Reply:
I agree with the common operation concern, but for that case I would prefer introducing an operator `??` (I think abusing `or` would be harder to teach, but I am not sure). For an actually named function, the name should be expressive.

Reply:
We decided to stick with this name for now and reconsider our options (e.g., dedicated syntax) some time later.